src/test/Components/CreateImageWizardV2/CreateImageWizard.test.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+
+import '@testing-library/jest-dom';
+
+import { screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import CreateImageWizard from '../../../Components/CreateImageWizardV2/CreateImageWizard';
+import ShareImageModal from '../../../Components/ShareImageModal/ShareImageModal';
+import { server } from '../../mocks/server.js';
+import {
+  clickNext,
+  renderCustomRoutesWithReduxRouter,
+  verifyCancelButton,
+} from '../../testUtils';
+
+const routes = [
+  {
+    path: 'insights/image-builder/*',
+    element: <div />,
+  },
+  {
+    path: 'insights/image-builder/imagewizard/:composeId?',
+    element: <CreateImageWizard />,
+  },
+  {
+    path: 'insights/image-builder/share /:composeId',
+    element: <ShareImageModal />,
+  },
+];
+
+// Mocking getComposes is necessary because in many tests we call navigate()
+// to navigate to the images table (via useNavigate hook), which will in turn
+// result in a call to getComposes. If it is not mocked, tests fail due to MSW
+// being unable to resolve that endpoint.
+// jest
+//   .spyOn(api, 'getComposes')
+//   .mockImplementation(() => Promise.resolve(mockComposesEmpty));
```

Comment:
This seems to be a leftover from the times when we used jest's `mockImplementation` instead of msw. You can remove the comment or we can clean it up in a follow up PR as it's not directly connected to this one. Both works for me.